.vscode/extensions.json

```diff
@@ -5,6 +5,7 @@
   // List of extensions which should be recommended for users of this workspace.
```

Comment:
Are comments allowed in json?

Reply:
In this file specifically, VS Code is okay with it

Reply:
Weird, ok!